system/includes/start.php

```diff
@@ -37,11 +39,10 @@
  * I think heavy core errors shouldnt be shown on the page and all others should be catched by an exception handler.
  */
 function handleShutdown() {
-  global $ajax;
   $error = error_get_last();
-  if (!$ajax && $error !== NULL) { // no Error notizes on ajax requests!
-    $info = "[ERROR] file:" . $error['file'] . ":" . $error['line'] . " <br/><i>" . $error['message'] . '</i>' . PHP_EOL;
-    echo '<div class="alert alert-error">' . $info . '</div>';
+  if (isset($error) && $error['type'] !== E_DEPRECATED) {
+    $info = $error['file'] . ' (line ' . $error['line'] . '): ' . $error['message'];
+	header('X-CT-Error: ' . $info);
```

Comment:
@milux, thanks for the patch!
A little nitpick here: please make indent better as well as you don't need to create `$info` var, but just put all that stuff into the header right away.